superset/views/core.py

```diff
@@ -1437,6 +1437,7 @@ def warm_up_cache(  # pylint: disable=too-many-locals,no-self-use
         dashboard_id = request.args.get("dashboard_id")
         table_name = request.args.get("table_name")
         db_name = request.args.get("db_name")
+        extra_filters = request.args.get("extra_filters")
```

Comment:
it would be great if you could add comment section on how would it be possible to fetch extra_filters for the dashboards / charts